src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -268,6 +268,40 @@ def find_manifest_file(deploy_root: Path) -> Path:
     )
 
 
+def find_and_read_manifest_file(deploy_root: Path) -> Any:
```

Comment:
why `-> Any`? This utility function should be opinionated about what it returns, or take arguments to control str vs bytes for example.